frontend/src/app/dashboard/dashboard.component.ts

```diff
@@ -153,6 +156,9 @@ export class DashboardComponent implements OnInit {
     this.transactions$ = this.stateService.transactions$
       .pipe(
         scan((acc, tx) => {
+          if (acc.find((t) => t.txid == tx.txid)) {
```

Comment:
What does this fix again? Was there a problem where transaction IDs got duplicated in the latest transactions list? Why would there be duplicate txids?